packages/hoppscotch-sh-admin/src/components/users/SharedRequests.vue

```diff
@@ -1,16 +1,16 @@
 <template>
-  <div>
+  <div class="px-4">
     <div v-if="fetching" class="flex justify-center">
       <HoppSmartSpinner />
     </div>
 
     <div v-else-if="error">{{ t('shared_requests.load_list_error') }}</div>
 
-    <div v-else-if="sharedRequests.length === 0" class="ml-3 mt-5 text-lg">
+    <div v-else-if="sharedRequests.length === 0" class="mt-5">
       {{ t('users.no_shared_requests') }}
     </div>
 
-    <div v-else class="mt-10">
```

Comment:
Could you elaborate on the need for the change in margin here?

Reply:
I want to keep the top margin similar to the users details tab and aligned to the start of the profile picture. Hence made the margin change